src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCNV.java

```diff
@@ -1,9 +1,9 @@
 package org.broadinstitute.hellbender.tools.exome;
 
 import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.barclay.argparser.Argument;
 import org.broadinstitute.hdf5.HDF5Library;
-import org.broadinstitute.hellbender.cmdline.Argument;
-import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
```

Comment:
Similar to above I wont comment on every one; but there are quite a few of these where the Barclay imports are not grouped together.